WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemComponentsScreen.kt

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.designsystem
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.R
+
+@Composable
+fun DesignSystemComponentsScreen(
+    modifier: Modifier = Modifier
+) {
+    LazyColumn (
+        modifier = modifier,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(
+            dimensionResource(id = R.dimen.reader_follow_sheet_button_margin_top)
+        )
+    ) {
+        item {
+            DesignSystemDataSource.componentsScreenButtonOptions.forEach { item ->
+                SelectOptionButton(
```

Comment:
Probably, need to replace these buttons with a simple TextView for list item, since it is just a list view now.